frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -33,19 +32,60 @@ const TitleSection = styled.div`
 `
 const InfoSection = styled.div`
     display: flex;
-    align-content: start;
-    align-items: flex-end;
-    gap: 1.2rem;
+    flex-wrap: wrap;
+    gap: 8px;
+    max-width: 950px;
+`
+const StyledCard = styled(Card)`
+    display: flex;
+    flex: 1 0 1;
+    padding: 8px 16px;
+    flex-direction: row;
+    gap: 4px;
+    max-width: 280px;
+    height: 60px;
+    background: ${tokens.colors.ui.background__light.hex};
+    gap: 24px;
+`
+
+const StyledTitleText = styled.div`
+    display: grid;
+    grid-direction: column;
+`
+
+const StyledTypography = styled(Typography)`
+    font-family: Equinor;
+    font-size: 32px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 40px; /* 125% */
+
+    @media (max-width: 500px) {
+        font-family: Equinor;
+        font-size: 16px;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 20px; /* 125% */
+    }
 `
 
-const HeaderText = (text: string) => {
+const HeaderText = (title: string, text: string) => {
     return (
-        <Typography variant="body_short" group="paragraph" color={tokens.colors.text.static_icons__secondary.hex}>
-            {text}
-        </Typography>
+        <StyledTitleText>
+            <Typography variant="meta" color={tokens.colors.text.static_icons__secondary.hex}>
+                {title}
+            </Typography>
+            <Typography variant="caption" color={tokens.colors.text.static_icons__secondary.hex}>
+                {text}
+            </Typography>
+        </StyledTitleText>
     )
 }
 
+const CardContent = (Element: JSX.Element) => {
```

Comment:
suggest renaming as it is the entire card and not the card content

Reply:
Why not just use the style directly instead of a function when the input mostly is wrapped in <></> anyways